java/src/test/java/testLinkedList/testLinkedList.java

```diff
@@ -0,0 +1,95 @@
+package testLinkedList;
+
+import LinkedList.LinkedList;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+public class testLinkedList {
+
+    @Test
+    public void testInstantiate() {

```

Comment:
Include comments in your tests, or messages in your assert statements for what you are expecting the test to do.